service/prometheus/prometheus.go

```diff
@@ -101,6 +115,18 @@ var (
 	// Kubelet IP (including port), and capture the IP.
 	KubeletPortRegexp = config.MustNewRegexp(`(.*):10250`)
 
+	// KubeSystemRegexp is the regular expression to match against the kube-system namespace.
+	KubeSystemRegexp = config.MustNewRegexp(`kube-system`)
```

Comment:
The `chart-operator` will start using the `giantswarm` namespace soon. So let's add this already here. @fgimenez @rossf7 correct?

Reply:
good shout, will do